sv.c

```diff
@@ -14106,6 +14106,7 @@ Perl_dirp_dup(pTHX_ DIR *const dp, CLONE_PARAMS *const param)
 #endif
 
     PERL_UNUSED_CONTEXT;
+    PERL_UNUSED_VAR(param);
```

Comment:
technically, shouldn't this be  `PERL_UNUSED_ARG`

Reply:
I'm not sure why that distinction even exists, but I'll change it.